wordpress/system/ksp-wordpress-packages-process-block.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-wordpress-packages-process-block
+spec:  
+  message: "(Alert) someone tried to install packages"
```

Comment:
Change the message to something else, and also decrease the severity, since it can trigger this alert if some even do the update also

Reply:
done